theano/gof/tests/test_vm.py

```diff
@@ -193,6 +193,15 @@ def time_linker(name, linker):
         time_linker('vmLinker_C', lambda: vm.VM_Linker(allow_gc=False,
                                                        use_cloop=True))
 
+def test_partial_function():
+    def callback(node, thunk, storage_map, compute_map): # to run as Stack
+        pass
+
+    x = tensor.scalar('input')
+    y = x ** 2
+    f = theano.function([x], [y + 7, y - 9, y / 14.], mode=Mode(optimizer=None,
+        linker=vm.VM_Linker(callback=callback)))
```

Comment:
add the parameter use_cloop=False for now. Otherwise, it could call the cvm. You can remove callback and use lazy=True to force the Stack implementation.